packages/core/src/components/Alert/Alert.jsx

```diff
@@ -29,11 +46,13 @@ export class Alert extends React.PureComponent {
 }
 
```

Comment:
I've been spot checking locally, and wanted to get your thoughts on adding a default role of "region" to the Alert component. Users can always override it with `role="alert"` or `role="alertdialog"` but having a default would ensure it always gets into the Landmarks menu for screen readers. Pairing this role with the always available ID + aria-labelledby makes it a named region that can be skipped to easily.

```javascript
Alert.defaultProps = {
  role: 'region'
};
```

Reply:
I think this is a smart default